cfme/cloud/instance/image.py

```diff
@@ -46,127 +133,100 @@ def __init__(self, name, provider, template_name=None, appliance=None):
-
-        # use accordion
-        # If a filter was applied, it will persist through navigation and needs to be cleared
-        if sel.is_displayed(search_box.clear_advanced_search):
-            logger.debug('Clearing advanced search filter')
-            sel.click(search_box.clear_advanced_search)
-        accordion.tree('Images', 'All Images')
+        self.view.sidebar.images.tree.click_path('All Images')
 
 
 @navigator.register(Image, 'AllForProvider')
 class ImageAllForProvider(CFMENavigateStep):
+    VIEW = ImageProviderAllView
     prerequisite = NavigateToAttribute('appliance.server', 'LoggedIn')
 
-    def am_i_here(self):
-        return match_page(summary='Images under Provider "{}"'.format(self.obj.provider.name))
-
     def step(self, *args, **kwargs):
         self.prerequisite_view.navigation.select('Compute', 'Clouds', 'Instances')
-
-        # use accordion
-        accordion.tree('Images by Provider', 'Images by Provider', self.obj.provider.name)
+        self.view.sidebar.images_by_provider.tree.click_path('Images by Provider',
+                                                             self.obj.provider.name)
 
 
 @navigator.register(Image, 'Details')
 class ImageDetails(CFMENavigateStep):
-    prerequisite = NavigateToSibling('All')
-
-    def am_i_here(self):
-        return details_page_check(name=self.obj.name, provider=self.obj.provider)
+    VIEW = ImageDetailsView
+    prerequisite = NavigateToSibling('AllForProvider')
 
     def step(self):
-        # Use list view to match name and provider
-        tb.select('List View')
-        cell = {'Name': self.obj.name, 'Provider': self.obj.provider.name}
+        self.prerequisite_view.toolbar.view_selector.select('List View')
         try:
-            sel.click(list_table.find_row_by_cell_on_all_pages(cell))
-        except NoSuchElementException:
-            raise ImageNotFound('Could not find image matching {}'.format(cell))
+            row = self.prerequisite_view.entities.get_entity(by_name=self.obj.name, surf_pages=True)
+        except ItemNotFound:
+            raise ImageNotFound('Failed to locate image with name "{}"'.format(self.obj.name))
+        row.click()
 
 
 @navigator.register(Image, 'ProvisionImage')
 class ImageProvisionImage(CFMENavigateStep):
+    VIEW = ImageProvisionView
     prerequisite = NavigateToSibling('Details')
 
-    # No am_i_here, page identifiers are not unique
-
     def step(self, *args, **kwargs):
-        life_btn('Provision Instances using this Image')
+        self.prerequisite_view.toolbar.lifecycle.item_select('Provision Instances using this Image')
 
 
-@navigator.register(Image, 'EditSelected')
+@navigator.register(Image, 'Edit')
 class ImageEdit(CFMENavigateStep):
+    VIEW = EditView
     prerequisite = NavigateToSibling('Details')
 
     def step(self, *args, **kwargs):
-        cfg_btn('Edit this Image')
-
-
-@navigator.register(Image, 'EditSelected')
-class ImageEditSelected(CFMENavigateStep):
```

Comment:
This isn't needed at the moment, and should be implemented for the collection anyway, supporting multi-select.